test/e2e/iam/aws_iam_policy.jsonc

```diff
@@ -91,6 +91,20 @@
                 "arn:aws:dynamodb:*:043455440429:table/e2e-*"
             ]
         },
+        {
+            "Sid": "E2EFrameworkS3",
+            "Effect": "Allow",
+            "Action": [
+                "s3:CreateBucket",
+                "s3:GetObject",
+                "s3:ListObjects",
```

Comment:
I'm seeing a warning for that particular action:

![image](https://user-images.githubusercontent.com/3299086/150397795-2a23bf0e-440a-4362-8228-c64ac4b416cb.png)

The doc says:

> To use this action in an AWS Identity and Access Management (IAM) policy, you must have permissions to perform the `s3:ListBucket` action.
> -- https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html